PURL-SPECIFICATION.rst

```diff
@@ -131,12 +131,11 @@ The rules for each component are:
 
 - **type**:
 
-  - The package ``type`` is composed only of ASCII letters and numbers, '.', '+'
-    and '-' (period, plus, and dash)
-  - The ``type`` cannot start with a number
-  - The ``type`` cannot contain spaces
-  - The ``type`` must NOT be percent-encoded
-  - The ``type`` is case insensitive. The canonical form is lowercase
+  - The package ``type`` MUST be composed only of ASCII letters and numbers,
+    '.', '+' and '-' (period, plus, and dash).
+  - The ``type`` MUST start with an ASCII letter.
+  - The ``type`` MUST NOT be percent-encoded.
+  - The ``type`` is case insensitive. The canonical form is lowercase.
```

Comment:
we have the canonicalization in the test suite already:
https://github.com/package-url/purl-spec/blob/17a74ea70c8c0329a1a742061e6101019405af82/test-suite-data.json#L39-L48